static/app/utils/theme/theme.chonk.tsx

```diff
@@ -0,0 +1,187 @@
+import type {SentryTheme} from './theme';
+
+const chonkShared = {
+  space: {
+    // @TODO(jonasbadalic): none doesn't need to exist
+    // none: 0,
+    subMicroscoptic: 1,
+    microscoptic: 2,
+    mini: 4,
+    small: 6,
+    medium: 8,
+    large: 12,
+    huge: 16,
+  },
+  borderRadius: {
```

Comment:
We also have `borderRadius` usages that use percent values like `50%`. Should we have a `rounded` token too to represent those?

Reply:
There are 69 matches for `border-radius: 50%` in the codebase

Reply:
So yeah, I think we'll definitely need a new key:value pair for the cases your both describing

So let's add at the end of the object a new key, I'd suggest naming it `round` with a value of `50%`